docs/api/event.md

```diff
@@ -3,6 +3,15 @@ id: event
 title: Event
 ---
 
+:::info TypeScript Support
+
+An event object has an associated TypeScript type:
+
+- Type Name - `Event`
+- Package - `@resolve-js/core`
+
+:::
+
 An event is an object of the following structure:
 
```

Comment:
Line 15 below:

`A number that is incremented for each consequent event with the current aggregateId.`

See my comment in the command-handler.md section.

Reply:
I fixed the error and updated the comment line.

Reply:
Thank you.